cms/templates/partials/text-video-section.html

```diff
@@ -26,15 +26,14 @@ <h1>{{ page.title }}</h1>
     </div>
   </div>
 </div>
-<div class="light-box-video-container">
-    <div id="light-box">
+{% if page.video_url %}
+    <div class="light-box-video-holder hidden">
         {% embed page.video_url as youtube_video %}
         {% if youtube_video %}
-            <div class="tv-video youtube-video" id="tv-yt-light-box-video">{{ youtube_video }}</div>
+            <div class="tv-video youtube-video" id="tv-light-box-yt-video" data-href='{{ youtube_video }}'></div>
```

Comment:
Nit: please wrap `data-href` value in double quotes instead of single quotes

Reply:
I think, I added single quote because `iframe` attributes have the double-quotes inside. 

`<iframe allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="" frameborder="0" height="270" src="https://www.youtube.com/embed/jgSTtn1oc1Q?feature=oembed&rel=0&enablejsapi=1" width="480"></iframe>`